lib/MiqVm/MiqRhevmVm.rb

```diff
@@ -51,12 +52,17 @@ def getCfg(_snap = nil)
       storage_obj    = storage_domains_by_id[storage_id]
 
       file_path = file_path_for_storage_type(storage_obj, disk)
+      file_path_s = file_path.to_s
 
       tag = "scsi0:#{idx}"
       cfg_hash["#{tag}.present"]    = "true"
       cfg_hash["#{tag}.devicetype"] = "disk"
-      cfg_hash["#{tag}.filename"]   = file_path.to_s
+      cfg_hash["#{tag}.filename"]   = file_path_s
       cfg_hash["#{tag}.format"]     = disk.format
+
+      if storage_obj.storage.type == "ISCSI" && !lv_active?(file_path_s)
```

Comment:
I still think we need to account for other storage types like fibrechannel here, `file_path_for_storage_type` does this
```
    storage_type = storage_obj&.storage&.type

    # TODO: account for other storage types here.
    case storage_type
    when "nfs", "glusterfs"
      add_fs_mount(storage_obj)
      fs_file_path(storage_obj, disk)
    else
      lun_file_path(storage_obj, disk)
    end
```

Maybe we instead of checking storage.type == "something" we have a method like storage_type_file? and storage_type_block? to abstract this